be/src/storage/tablet.cpp

```diff
@@ -391,7 +391,7 @@ void Tablet::delete_expired_inc_rowsets() {
     std::vector<Version> expired_versions;
     std::unique_lock wrlock(_meta_lock);
     for (auto& rs_meta : _tablet_meta->all_inc_rs_metas()) {
-        double diff = ::difftime(now, rs_meta->creation_time());
+        int64_t diff = now - rs_meta->creation_time();
```

Comment:
why don't you use difftime?

Reply:
it returns double, it's a simple `-` operation(so unnecessary to use difftime), and someone has already misused it